src/env/node.js

```diff
@@ -63,14 +111,66 @@ export default {
 		process.env.NODE_ENV = "test";
 	},
 	done (result, options, event, root) {
-		let messages = root.toString({ format: options.format ?? "rich" });
-		let tree = getTree(messages).toString();
-		tree = format(tree);
-		logUpdate(tree);
+		makeCollapsible(root)
+		render(root, options);
 
 		if (root.stats.pending === 0) {
+
+			let hint = `
+<b>You are in interactive mode now.</b>
+Use <b>↑</b> and <b>↓</b> arrow keys to navigate groups of tests, <b>→</b> and <b>←</b> to expand and collapse them respectively.
+Press <b>^C</b> (<b>Ctrl+C</b>) or <b>q</b> to quit.
```

Comment:
```suggestion
Press <b>^C</b> (<b>Ctrl+C</b>) or <b>q</b> to quit interactive mode.
```
There, you could add it there.